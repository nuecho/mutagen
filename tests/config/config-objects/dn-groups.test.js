importDependencies("dn-dependencies-config.json");
importDependencies("dnGroups-dependencies-config.json");
const checkKey = (cfgObject) => {return cfgObject.group.name === "cfgObjectTest"};

cfgObjectTests("dnGroups", {checkKey: checkKey});
