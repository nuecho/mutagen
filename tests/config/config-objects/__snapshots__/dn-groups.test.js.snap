// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[mutagen dnGroups config import-export] should create the dnGroups: the dnGroups have been created 1`] = `
Object {
  "stderr": "",
  "stdout": "+ dngroup => Environment/cfgObjectTest
Completed. 1 object(s) imported.
",
}
`;

exports[`[mutagen dnGroups config import-export] should throw exception when new dnGroups to import miss mandatory properties dnGroups: exception has been thrown 1`] = `
Object {
  "stderr": "Mandatory properties [type] not set in dngroup 'Environment/emptyCfgObjectTest'
Cannot import configuration: some configuration objects' mandatory properties for creation are not set.
",
  "stdout": "",
}
`;

exports[`[mutagen dnGroups config import-export] should update the dnGroups: the dnGroups have been updated 1`] = `
Object {
  "stderr": "",
  "stdout": "~ dngroup => Environment/cfgObjectTest
Completed. 1 object(s) imported.
",
}
`;
